src/app/scss/components/_buttons.scss

```diff
@@ -2,11 +2,13 @@
 
 .base-button {
   background: var(--secondary);
+  min-width: 135px;
```

Comment:
is this a good idea? as it will change the width of all the buttons, are they still responsive and etc?